Feat(M&B): Notifications (#4897)
Kong/developer.konghq.com: app/metering-and-billing/subscriptions.md

Subscriptions follow a billing cycle determined by their related [rate card](/metering-and-billing/product-catalog/#rate-cards), anchored to one of the following:

* The subscription start date, either the creation date or a specified future date.
* The first day of the month, with usage prorated for the partial initial period.

To add a subscription to a customer, navigate to **{{site.metering_and_billing}}** > **Billing**, click your customer, and then click the **Subscriptions** tab in the {{site.konnect_short_name}} UI.

Active customer subscriptions can be enhanced with [add-ons](/metering-and-billing/add-ons/), which allow you to make changes to a customer's entitlements without changing the plan directly.

## Change plans

Changing a plan switches a customer to a completely different plan. This is equivalent to canceling the current subscription and starting a new one on the new plan, but without any interruption in service.

To change a customer's plan:

1. In the {{site.konnect_short_name}} sidebar, click **{{site.metering_and_billing}}**.
1. In the {{site.metering_and_billing}} sidebar, click **Billing**.
1. Click the customer whose subscription you want to change.

> 1. Click the **Subscriptions** tab.

Reviewer: ```suggestion
1. Click the **Subscription** tab.
```

1. Click **Manage**.
1. Select the new plan.
1. Click **Next**.

> 1. Choose when the plan change should be effective: **At the end of the billing period** or **Immediately**.

Reviewer: ```suggestion
1. Select when the plan change should be effective **At the end of the billing period** or **Immediately**.
   * **At the end of the billing period**: Select for downgrades to avoid disrupting the customer's current service period. 
   * **Immediately**: Select for upgrades so the customer gets access to the new plan right away.
  
```

1. Click **Save Changes**.

> Choose **At the end of the billing period** for downgrades to avoid disrupting the customer's current service period. Choose **Immediately** for upgrades so the customer gets access to the new plan right away.

Reviewer: ```suggestion

```

## Cancel a subscription

Canceling a subscription ends it and stops future invoices from being generated. You can cancel immediately or at the end of the current billing period.

To cancel a subscription:

1. In the {{site.konnect_short_name}} sidebar, click **{{site.metering_and_billing}}**.
1. In the {{site.metering_and_billing}} sidebar, click **Billing**.
1. Click the customer whose subscription you want to cancel.
1. Click the **Subscriptions** tab.
1. Click **Cancel Subscription**.

> 1. Choose when the cancellation should be effective: **At the end of the billing period** or **Immediately**.

Reviewer: ```suggestion
1. Select when the plan change should be effective **At the end of the billing period** or **Immediately**.
   * **At the end of the billing period**: Select for downgrades to avoid disrupting the customer's current service period. 
   * **Immediately**: Select for upgrades so the customer gets access to the new plan right away.
```

> 1. Click **Cancel Subscription**.

Reviewer: ```suggestion
1. Click **Save Changes**.
```
